src/Bicep.LangServer.IntegrationTests/Completions/SnippetTemplates/res-policy-definition-deploy-effect/main.combined.bicep

```diff
@@ -0,0 +1,87 @@
+  name: 'name'
+  properties: {
+    displayName: 'displayName'
+    policyType: 'Custom'
+    mode: 'All'
+    description: 'description'
+    metadata: {
+      version: '0.1.0'
+      category: 'category'
+      source: 'source'
+    }
+    parameters: {
+      parameterName: {
+        type: 'String'
+        defaultValue: 'defaultValue'
+        metadata: {
+          displayName: 'displayName'
+          description: 'description'
+        }
+      }
+    }
+    policyRule: {
+      if: {
+        allOf: [
+          {
+            field: 'field'
+            equals: 'conditionValue'
+          }
+        ]
+      }
+      then: {
+        effect: 'deployIfNotExists'
+        details: {
+          roleDefinitionIds: [
+            '/providers/Microsoft.Authorization/roleDefinitions/b24988ac-6180-42a0-ab88-20f7382dd24c'
+          ]
+          type: 'type'
+          existenceCondition: {
+            allOf: [
+              {
+                field: 'field'
+                equals: 'conditionValue'
+              }
+            ]
+          }
+          deployment: {
+            properties: {
+              mode: 'incremental'
+//@[19:100) [no-hardcoded-env-urls (Warning)] Environment URLs should not be hardcoded. Use the environment() function to ensure compatibility across clouds. Found this disallowed host: "management.azure.com" (CodeDescription: bicep core(https://aka.ms/bicep/linter/no-hardcoded-env-urls)) | 'incremental'\r\n              template: {\r\n                '$schema': 'https://sc|
```

Comment:
New linter hard at work :-)

Reply:
... and it looks like you found a bug.

Reply:
https://github.com/Azure/bicep/issues/2948

Reply:
Until this bug gets fixed, you can just leave this warning in main.combined.bicep for now.

Reply:
thanks Marcus - sure thing!